pkgs/applications/graphics/pinta/default.nix

```diff
@@ -1,14 +1,18 @@
-{ lib, fetchFromGitHub, buildDotnetPackage, dotnetPackages, gtksharp,
-  gettext }:
+{ lib
+, fetchFromGitHub
+, buildDotnetPackage
+, dotnetPackages
+, gtksharp
+, gettext
+}:
 
 let
   mono-addins = dotnetPackages.MonoAddins;
 in
 buildDotnetPackage rec {
-  name = "pinta-1.6";
-
```

Comment:
Maybe a dumb question because I don't know how `buildDotnetPackage` works, but where is the `pname` :thinking: ?

Reply:
`buildDotnetPackage` is older then `pname`. it is equivalent to `baseName`. This renaming will come in a future PR.